src/Ben.Demystifier/EnhancedStackTrace.Frames.cs

```diff
@@ -538,18 +531,8 @@ private static string GetPrefix(ParameterInfo parameter, Type parameterType)
 
         private static ResolvedParameter GetParameter(ParameterInfo parameter)
         {
+            var prefix = GetPrefix(parameter);
             var parameterType = parameter.ParameterType;
-            var prefix = GetPrefix(parameter, parameterType);
-
-            if (parameterType == null)
```

Comment:
Before anyone asks - `parameter.ParameterType` always returns a value. Therefore this code is heuristically never reachable.